apps/style/common.scss

```diff
@@ -938,32 +938,6 @@ input[type="radio"] {
   }
 }
 
-.modal-content {
-  .instructionsImage {
-    height: 35px;
-    vertical-align: text-bottom;
-  }
-  .instructions2 {
-    .instructionsImage {
-      height: 24px;
-      vertical-align: text-bottom;
-    }
-  }
-}
-
-#prompt-table {
-  .instructionsImage {
-    height: 25px;
-    vertical-align: text-bottom;
-  }
-  #prompt2 {
-    .instructionsImage {
-      height: 20px;
-      vertical-align: text-bottom;
-    }
-  }
-}
-
```

Comment:
instructionsImage and promp2 are both only used in Studio levels. Move the styles there (and refactor them some)